docs/make.jl

```diff
@@ -22,7 +22,7 @@ makedocs(;
         "Examples" => [
             "Code examples" => "code_example.md",
             "Markdown examples" => "markdown-examples.md",
-            "MIME output examples" => "mime_examples.md"
+            "MIME output examples" => "mime_examples.md",
             "Updating to DocumenterVitepress from Documenter example" => "updated_example.md"
```

Comment:
maybe a more descriptive name for the file, something like `up_docs_expample` or `update_documenter_docs_example` or.... something else, this will be the actual path, hence searching for it could be easier. 

Apologies for so many small details 😄.

Reply:
Sure, can it be `documenter_to_vitepress_docs_example.md`  @lazarusA  ?